spec/integration/integration_helper.rb

```diff
@@ -5,6 +5,7 @@
 begin
   require 'active_support'
 rescue LoadError
+  print 'No "active_support" loaded'
```

Comment:
cool.. lets not do this right now, as we didnt drop support yet.. but thanks @grobie for the info.